service/docs/source/security.rst

```diff
@@ -206,326 +107,99 @@ JSON data.
+
+The GEOPM Service’s access management system enables a system administrator to control which features can be accessed through the MSR driver. The service also prevents permanent changes to the MSR driver. As such, administrators may want to provide MSR access through the GEOPM Service to processes that lack the CAP_SYS_RAWIO Linux `capability <https://man7.org/linux/man-pages/man7/capabilities.7.html>`__.
+
+
+Intel Speed Select Device Driver
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-The GEOPM Service may be used as a gateway to the Intel Speed Select
-device driver which is loaded as the ``/dev/isst_interface``
-device. This device driver enables a wide range of capabilities
-introduced with the 3rd generation Xeon Scalable server processor.
+The GEOPM Service can act as a gateway to the Intel Speed Select device driver, loaded as the ``/dev/isst_interface`` device. This driver offers a broad set of capabilities introduced with the 3rd generation Xeon Scalable server processor.
 
 https://www.kernel.org/doc/html/latest/admin-guide/pm/intel-speed-select.html
 
-The specific features enabled through the GEOPM Service are the
-`SST-CP <https://www.kernel.org/doc/html/latest/admin-guide/pm/intel-speed-select.html#intel-r-speed-select-technology-core-power-intel-r-sst-cp>`__
-and
-`SST-TF <https://www.kernel.org/doc/html/latest/admin-guide/pm/intel-speed-select.html#intel-r-speed-select-technology-turbo-frequency-intel-r-sst-tf>`__
-features. Using the ``isst_interface`` device driver requires the
-Linux
-`capability <https://man7.org/linux/man-pages/man7/capabilities.7.html>`__
-of CAP_SYS_ADMIN because changes may impact system performance for
-other users of the system. The ISST interface may also be used to
-change the hardware characteristics reported by the Linux kernel, such
-as number of cores, base frequency, and which turbo frequencies are
-achievable.
-
-
-LevelZero sysman library interface
+The specific features supported through the GEOPM Service are the `SST-CP <https://www.kernel.org/doc/html/latest/admin-guide/pm/intel-speed-select.html#intel-r-speed-select-technology-core-power-intel-r-sst-cp>`__ and `SST-TF <https://www.kernel.org/doc/html/latest/admin-guide/pm/intel-speed-select.html#intel-r-speed-select-technology-turbo-frequency-intel-r-sst-tf>`__ features. Use of the ``isst_interface`` device driver necessitates the Linux `capability <https://man7.org/linux/man-pages/man7/capabilities.7.html>`__ of CAP_SYS_ADMIN because changes may influence system performance for other users. The ISST interface can also alter the hardware characteristics reported by the Linux kernel, including the number of cores, base frequency, and achievable turbo frequencies.
```

Comment:
```suggestion
The specific features supported through the GEOPM Service are the `SST-CP <https://www.kernel.org/doc/html/latest/admin-guide/pm/intel-speed-select.html#intel-r-speed-select-technology-core-power-intel-r-sst-cp>`__ and `SST-TF <https://www.kernel.org/doc/html/latest/admin-guide/pm/intel-speed-select.html#intel-r-speed-select-technology-turbo-frequency-intel-r-sst-tf>`__ features. Use of the ``isst_interface`` device driver necessitates the Linux `capability <https://man7.org/linux/man-pages/man7/capabilities.7.html>`__ of ``CAP_SYS_ADMIN`` because changes may influence system performance for other users. The ISST interface can also alter the hardware characteristics reported by the Linux kernel, including the number of cores, base frequency, and achievable turbo frequencies.
```